src/components/Person/Person.jsx

```diff
@@ -1 +1,47 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerType = (sex === 'm')
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {(isMarried)
+          ? `${partnerName} is my ${partnerType}`
+          : `I am not married`
+        }
```

Comment:
It's better to put this logic into an additional variable

Reply:
I disagree.
This logic is very simple and is not repeated anywhere else.